src/diffusers/loaders/unet.py

```diff
@@ -64,7 +64,12 @@ class UNet2DConditionLoadersMixin:
     unet_name = UNET_NAME
 
     @validate_hf_hub_args
-    def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict[str, torch.Tensor]], **kwargs):
+    def load_attn_procs(
```

Comment:
Since this method will be deprecated, I think it's okay to not propagate these changes:
https://github.com/huggingface/diffusers/blob/464374fb87610c53b2cf81e08d3df628fada3ce4/tests/models/unets/test_models_unet_2d_condition.py#L1080